src/System.Private.CoreLib/src/System/Threading/ThreadPool.cs

```diff
@@ -31,6 +31,7 @@
 using System.Diagnostics;
 using System.Diagnostics.Contracts;
 using System.Runtime.InteropServices;
+using System.Collections.Concurrent;
```

Comment:
There are still a fair number of differences. Eventually it would be best to put most of the code in the shared folder but for now I think it would be easiest to eliminate all of the insignificant differences and only keep the differences that were intentionally made on the CoreRT side.